package.json

```diff
@@ -27,6 +27,7 @@
     "express": "4.17.1",
     "json-stable-stringify": "1.0.1",
     "loglevel": "1.6.2",
+    "node-cache": "^4.2.0",
```

Comment:
Is node-cache usable in a browser, or just node.js?

Reply:
it is usable